### PR TITLE
buit the search-bar component white sub-components as the dropdown, b…

### PR DESCRIPTION
**Pull request by samynaj**
…utton and search input and style all in the main component sass file
Screenshot (42)
image
image
worked on the feedback

### CODE REVIEW

**Review by Midetech — COMMENTED** [review]
Yo Man, Nice work here.
Some of the few feedback I have.
1. The commented code should be removed if there are not needed and if you think its there for a purpose then you should give it a comment that explain its purpose so anyone using the component can figure it out.

2. If I am right and there's a way around it. You could spilt the component into two as in SearchBar a component and FIndFood a component and pass it inside each other.

3. The styling from my end here seem not to have margin at the right and left.
image

4. I was a bit confused by the component name oooo it should be **index.jsx** instead of **search-bar.component.jsx.**

my thoughts tho.✌✌✌

**Review by Mlg29 — CHANGES_REQUESTED** [review]
Kindly take a look at what Femi said about your pr, I believe that's the only thing u need to work on now

File: dae-alright/src/components/search-bar/component/button/button.component.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+// import './button.style.scss';
```
Comment (SEUNAGBEYE) [inline dae-alright/src/components/search-bar/component/button/button.component.jsx]: Please remove this

File: dae-alright/src/components/search-bar/component/dropdown/search.dropdown.component.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from  'react';
+
+
+const SearchDropDown = ({places, text}) => (
+    <select className='search-drop' >
+        <option className='option' key='0' value="">{text}</option>
```
Comment (SEUNAGBEYE) [inline dae-alright/src/components/search-bar/component/dropdown/search.dropdown.component.jsx]: You don't need a key here

File: dae-alright/src/components/search-bar/search-bar.component.jsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        }
+    }
+
+   handleClick() {
+       this.setState({
+           isFindFood: true
+       })
+   }
+
+    // handleChangeSearch(props) {
```
Comment (SEUNAGBEYE) [inline dae-alright/src/components/search-bar/search-bar.component.jsx]: Remove this commented code if not needed

**Review by SEUNAGBEYE — CHANGES_REQUESTED** [review]

**Comment by SEUNAGBEYE** [comment]
From my review, I can say you are yet to complete the feedback on your PR. Also, I do not agree with your naming style. 
Rename the following `folders/files`
`search-bar` -> `SearchBar`
`component` -> `components`
`button` -> `Button`
`dropdown` -> `DropDown`
`button.component.jsx` -> `index.js`
`search.dropdown.component.jsx` -> `index.js`
`search-bar.component.jsx` -> `index.js`



Components with a default `export` in a folder should be named as `index.js`

**Comment by SEUNAGBEYE** [comment]
All styles within a component folder too should be renamed to `styles.scss`

File: dae-alright/src/components/search-bar/component/button/button.component.jsx (hunk 1 above)
Comment (LuthfulahiO) [reply dae-alright/src/components/search-bar/component/button/button.component.jsx]: @samynaj why haven't made resolved this?

File: dae-alright/src/components/search-bar/component/dropdown/search.dropdown.component.jsx (hunk 2 above)
Comment (LuthfulahiO) [reply dae-alright/src/components/search-bar/component/dropdown/search.dropdown.component.jsx]: This too?

File: dae-alright/src/components/search-bar/search-bar.component.jsx (hunk 3 above)
Comment (LuthfulahiO) [reply dae-alright/src/components/search-bar/search-bar.component.jsx]: @samynaj ooooooo

**Pull request merged**